compose.js

```diff
@@ -89,7 +89,11 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
     if (!Array.isArray(dstDescriptor.initializers)) dstDescriptor.initializers = [];
-    dstDescriptor.initializers.push.apply(dstDescriptor.initializers, srcDescriptor.initializers.filter(isFunction));
+    srcDescriptor.initializers.forEach(i => {
```

Comment:
Fair enough. Let me change the code to the `reduce` implementation so that you can merge it. :)